.pipelines/cni/load-test-templates/restart-hns-template.yaml

```diff
@@ -0,0 +1,29 @@
+parameters:
+  clusterName: ""
+  cni: "cniv1"
+
+steps:
+  - task: AzureCLI@1
+    inputs:
+      azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+        make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}
+        kubectl get pod -owide -A
+
+        echo "Validate pod IP assignment before HNS restart"
+        make test-validate-state CNI_TYPE=${{ parameters.cni }} OS_TYPE=windows
+
+        podList=`kubectl get pods -n kube-system -l os=windows -owide --no-headers | awk '{print $1}'`
```

Comment:
This should include label for privileged pod too right ?

Reply:
Yes, I didn't realize that our noop deployment also has this same label. Pairing the labels should prevent any edge cases in the future.